.gitignore

```diff
@@ -9,9 +9,13 @@ __pycache__
 *.pyd
 *.bak
 *.c
+skimage/feature/_haar.cpp
```

Comment:
Did you also have to build this file manually? I had to do so on my machine, but it appeared to be working on Travis so I thought it was just a quirk of my own install...!

Reply:
no, i just did `python setup.py develop`. I didn't know if we had other cpp code. so I didn't want to blanket ignore them.